include/deal.II/integrators/patches.h

```diff
@@ -38,9 +38,9 @@ namespace LocalIntegrators
   {
     template <int dim>
     inline void
-      points_and_values(Table<2, double> &                          result,
-                        const FEValuesBase<dim> &                   fe,
-                        const ArrayView<const std::vector<double>> &input)
+    points_and_values(Table<2, double> &                          result,
+                      const FEValuesBase<dim> &                   fe,
+                      const ArrayView<const std::vector<double>> &input)
```

Comment:
I am a bit dissapointed that clang-format still cannot align the `&` properly :-(